Dockerfile

```diff
@@ -60,17 +84,29 @@ RUN apk add -u shadow libc6-compat && \
 
 USER secrets-provider
 
-COPY --from=secrets-provider-builder /opt/secrets-provider-for-k8s/secrets-provider /usr/local/bin/
+# =================== RELEASE MAIN CONTAINER ===================
+FROM secrets-provider-base as secrets-provider
 
-ENTRYPOINT [ "/usr/local/bin/secrets-provider"]
+COPY --from=secrets-provider-builder /opt/secrets-provider/secrets-provider /usr/local/bin/
+
+CMD [ "/usr/local/bin/secrets-provider"]
+
+# =================== DEBUG MAIN CONTAINER ===================
+FROM secrets-provider-base as secrets-provider-debug
+
+COPY --from=secrets-provider-builder-debug /go/bin/dlv /usr/local/bin/
+
```

Comment:
you can group the COPYs together IMO

Reply:
We don't do it for any other consequent COPY in the file.